### PR TITLE
Improve tooltips integration

### PR DESCRIPTION
**Pull request by jaco3n4**

### CODE REVIEW

**Comment by mathieugerard** [comment]
Visually it's nice.

- [ ] Strings should be translated based on the locale
- [ ] Need to be tested on mobile phones to make sure they don't appear
- [ ] Need to be tested on multiple browsers including IE11
- [ ] Could we have tooltip on languages and universe buttons ("Change venue language" and "Change venue universe")

**Comment by mathieugerard** [comment]
PT translation:

Menu: Menu
Directions: Direções
Zoom in: Aumentar o zoom
Zoom out: Diminuir o zoom
Reset bearing to north<br>Use ctrl+drag to rotate the map: Orientar em direção ao norte<br>Utilize ctrl+arrastar para orientar o mapa.
Change venue language: Alterar a lingua
Change venue universe: Alterar a visualisation


FR translation:

Menu: Menu
Directions: Directions
Zoom in: Augmenter le zoom
Zoom out: Diminuer le zoom
Reset bearing to north<br>Use ctrl+drag to rotate the map: Orienter au nord<br>Utiliser ctrl+déplacer pour tourner la carte
Change venue language: Changer la langue du lieu
Change venue universe: Changer l'univers du lieu

**Comment by mathieugerard** [comment]
--- NL ---

Menu: Menu
Directions: Richtingen
Zoom in: Schaal vergroten
Zoom out: Schaal verkleinen
Zet de peiling terug naar het noorden
Use Ctrl+drag to rotate the map: Naar het noorden richten
Gebruik Ctrl+Bewegen om de kaart te draaien
Change venue language: Taal van de plaats wijzigen
Change venue universe: Universum van de plaats wijzigen


--- DE---

Menu: Menü
Directions: Richtungen
Zoom in: vergrößern
Zoom out: verkleinern
Peilung nach Norden zurücksetzen
Use ctrl+drag to rotate the map: Nach Norden ausrichten
Nutzen Sie die Tasten Strg + Bewegen um die Karte auszurichten
Change venue language: Sprache des Veranstaltungsortes ändern
Change venue universe: Universum des Veranstaltungsortes ändern

File: src/controls/navigationControl/navigationControl.ts [hunk 1]
```diff
@@ -20,36 +22,48 @@ class NavigationControl {
...
       this._compass.appendChild(this._compassArrow);
+
+      this.refreshLocale()
+    }
+  }
+
+  public refreshLocale () {
+    if (!$(this._container).hasClass('mwz-small')) {
+      setTimeout(() => {
+        $('.mapboxgl-ctrl-zoom-in').attr('data-original-title', translate('zoom_in'))
```
Comment (Sparksx) [inline src/controls/navigationControl/navigationControl.ts]:
```suggestion
        $(this._container).find('.mapboxgl-ctrl-zoom-in').attr('data-original-title', translate('zoom_in'))
```

File: src/controls/navigationControl/navigationControl.ts [hunk 2]
```diff
@@ -20,36 +22,48 @@ class NavigationControl {
...
+        $('.mapboxgl-ctrl-zoom-out').attr('data-original-title', translate('zoom_out'))
```
Comment (Sparksx) [inline src/controls/navigationControl/navigationControl.ts]:
```suggestion
        $(this._container).find('.mapboxgl-ctrl-zoom-out').attr('data-original-title', translate('zoom_out'))
```

File: src/controls/navigationControl/navigationControl.ts [hunk 3]
```diff
@@ -20,36 +22,48 @@ class NavigationControl {
...
+        $('.mapboxgl-ctrl-compass').attr('data-original-title', translate('reset_bearing_to_north') + '<br>' + translate('use_ctrl'))
```
Comment (Sparksx) [inline src/controls/navigationControl/navigationControl.ts]:
```suggestion
        $(this._container).find('.mapboxgl-ctrl-compass').attr('data-original-title', translate('reset_bearing_to_north') + '<br>' + translate('use_ctrl'))
```

File: src/header/searchBar/searchBar.ts [hunk 4]
```diff
@@ -86,6 +96,16 @@ export class SearchBar extends DefaultControl {
     }
   }
 
+  public refreshTooltips () {
+    if (!$(this.map._container).hasClass('mwz-small')) {
+      setTimeout(() => {
+        this._container.find('#mwz-menu-button').attr('data-original-title', translate('menu'))
+        this._container.find('#mwz-header-directions-button').attr('data-original-title', translate('directions'))
+        $('[data-toggle="tooltip"]').tooltip({ trigger: 'hover' })
```
Comment (Sparksx) [inline src/header/searchBar/searchBar.ts]:
```suggestion
        this._container.find('[data-toggle="tooltip"]').tooltip({ trigger: 'hover' })
```

**Comment by mathieugerard** [comment]
IMG_3013BD024695-1
Tooltip remains displayed on mobile. Actually on mobile it should not be displayed at all.

**Pull request merged**